Cargo.toml

```diff
@@ -22,20 +22,16 @@ default = [
     "bevy_gltf",
     "bevy_wgpu",
     "bevy_winit",
+    "bevy_dynload",
     "render",
-    "dynamic_plugins",
```

Comment:
We should also update the feature name in https://github.com/bevyengine/bevy/blob/master/docs/cargo_features.md

Reply:
Yup, and I also updated publish.sh